docs/references/javascript/organization/organization.mdx

```diff
@@ -42,31 +42,173 @@ function update(params: UpdateOrganizationParams): Promise<Organization>;
 | `name` | `string` | The organization name. |
 | `slug?` | `string \| undefined` | The organization slug. |
 
-#### Returns
+#### `update()` returns
 
 | Type | Description |
 | --- | --- |
-| <code>Promise\<[Organization][org-ref]\></code> | This method returns a `Promise` which resolves to an [Organization][org-ref]. |
+| `Promise<Organization>` | This method returns a `Promise` which resolves to an [`Organization`](#organization) object. |
+
+#### `update()` example
+
+<InjectKeys>
+
+<CodeBlockTabs type="npm-script" options={['NPM module', '<script>']}>
+```js filename="main.js"
+import Clerk from '@clerk/clerk-js';
+
+// Initialize Clerk with your Clerk publishable key
+const clerk = new Clerk('{{pub_key}}');
+await clerk.load();
+
+if (clerk.user) {
+  // Get the current organization
+  const currentOrganization = clerk.organization;
+
+  // Update the organization's name
+  currentOrganization.update({ name: "New Name" })
+    .then((res) => console.log(res))
+    .catch((error) => console.log("An error occurred:", error.errors));
+} else {
+  document.getElementById("app").innerHTML = `
+    <div id="sign-in"></div>
+  `;
+
+  const signInDiv = document.getElementById("sign-in");
+
+  clerk.mountSignIn(signInDiv);
+}
+```
+
+```html filename="index.html"
+  <div id="app"></div>
+
+  <!-- Initialize Clerk with your
+  Clerk publishable key and Frontend API URL -->
+  <script
+    async
+    crossorigin="anonymous"
+    data-clerk-publishable-key='{{pub_key}}'
+    src="https://[your-domain].clerk.accounts.dev/npm/@clerk/clerk-js@latest/dist/clerk.browser.js"
+    type="text/javascript"
+  ></script>
+
+  <script>
+    window.addEventListener("load", async function () {
+      await Clerk.load();
+      if (Clerk.user) {
+        // Get the current organization
+        const currentOrganization = Clerk.organization;
+
+        // Update the organization's name
+        currentOrganization.update({ name: "New Name" })
```

Comment:
Don't forget to add code checking if Clerk.organization exists to these examples as well 👍